File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_2/Fibonacci.java
public class Fibonacci {
        public static int[] generateFibonacciNumbers(int n) {
            // Write your code here.
            int[] arr = new int[n];
            arr[0]=0;
            if(n>1)
                arr[1]=1;
            for (int i =2; i<n; i++){
                arr[i]=arr[i-1]+arr[i-2];
            }
            return arr;
        }
}

File: SecondYear/Junior-Hawk_ArchitAgarwal_2226cseai1186_2/Week_2/fibonacci.cpp
vector<int> generateFibonacciNumbers(int n) {
    if(n==1){
        return {0};
    }
    else if(n==2){
        return {0,1};
    }
    vector<int> ans(n,0);
    ans[1]=1;
    for(int i=2;i<n;i++){
        ans[i] = ans[i-1]+ans[i-2];
    }
    return ans;
}